scripts/util/players.py

```diff
@@ -0,0 +1,212 @@
+from ruamel.yaml import YAML
+
+from .io import Importable, Exportable, JsonSerializable
+
+from .index import Indexable
+
+from .error import MissingKeyError, InvalidCountryCodeError
+
+from . import INDEX_LIST_SUFFIX
+
+import pycountry
+
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Player(JsonSerializable):
+
+    def __init__(self, name, liquipedia=None, twitch=None, youtube=None,
+
+                 facebook_gaming=None, team=None):
+
+        self.name = name
+
+        self.liquipedia = liquipedia
+
+        self.youtube = youtube
+
+        self.twitch = twitch
+
+        self.facebook_gaming = facebook_gaming
+        self.team = team
+
+
+class PlayerList(Player, Indexable, Importable, Exportable, JsonSerializable):
+
+    errors = []
+    id_list = []
+    indexed_keys = ["id", "name", "aoeelo", "esportsearnings", "liquipedia",
+                    "country"]
+    # Don't throw an exeption for these keys if they are missing
+    voluntary_keys = ["aoeelo", "esportsearnings", "liquipedia", "country"]
+
+    # We don't check for duplicates here
+    non_unique_keys = ["country"]
```

Comment:
Refactor those three to a tuple, e.g. `("id", False, False)` -> `(key, voluntary, non-unique)`